[CI] redmine master version tests failed

**Problem**
redmine master version tests failed with the following bundle install error.
https://github.com/gtt-project/redmine_gtt/actions/runs/7258962536/job/19775204289?pr=270
```
Bundler could not find compatible versions for gem "activerecord":
  In Gemfile:
activerecord-postgis-adapter (~> 7.1.1) was resolved to 7.1.1, which depends
on
      activerecord (~> 6.1)

    rails (= 7.1.2) was resolved to 7.1.2, which depends on
      activerecord (= 7.1.2)
Error: Process completed with exit code 6.
```

**Expectation**
Gem installation and tests should be passed.

--- a/.github/workflows/test-postgis.yml
+++ b/.github/workflows/test-postgis.yml
@@ -87,14 +87,13 @@ jobs:
               encoding: utf8
           EOF
 
-      # - name: Adjust Gem environment
-      #  run: |
-      #    case "${{ matrix.redmine_version }}" in
-      #      4.2-stable)
-      #        echo "GEM_RGEO_ACTIVERECORD_VERSION=6.2.2" >> ${GITHUB_ENV}
-      #        echo "GEM_ACTIVERECORD_POSTGIS_ADAPTER_VERSION=5.2.3" >> ${GITHUB_ENV}
-      #        ;;
-      #    esac
+      - name: Adjust Gem environment
+        run: |
+          case "${{ matrix.redmine_version }}" in
+            master)
+              echo "GEM_ACTIVERECORD_POSTGIS_ADAPTER_VERSION=9.0.1" >> ${GITHUB_ENV}
+              ;;
+          esac
 
       - name: Install Ruby dependencies
         working-directory: redmine

